dockerfiles/publish_manylinux/Dockerfile

```diff
@@ -0,0 +1,92 @@
+    && bash Miniconda3-latest-Linux-x86_64.sh -b -p /opt/miniconda \
+    && rm -f Miniconda3-latest-Linux-x86_64.sh
+
+# Prepare shared env
+RUN . /opt/miniconda/etc/profile.d/conda.sh && \
+    conda activate base && \
+    pip install conan==1.51.1 && \
+    pip install twine && \
+    pip cache purge
+RUN . /opt/miniconda/etc/profile.d/conda.sh && \
+    conda activate base && \
+    conda install cmake && \
+    conda clean --all
+
+# gmake[2]: *** No rule to make target `/usr/local/lib/libtiff.so', needed by `lib/xrprimer_cpp.cpython-36m-x86_64-linux-gnu.so'.  Stop.
+RUN ln -s /opt/miniconda/bin/conan /opt/rh/devtoolset-10/root/usr/bin/conan && \
+    ln -s /opt/miniconda/bin/twine /opt/rh/devtoolset-10/root/usr/bin/twine && \
+    ln -s /opt/miniconda/bin/cmake /opt/rh/devtoolset-10/root/usr/bin/cmake && \
+    ln -s /usr/lib64/libjpeg.so.62 /usr/local/lib64/libjpeg.so && \
+    ln -s /usr/lib64/libwebp.so.4 /usr/local/lib/libwebp.so && \
+    ln -s /usr/lib64/libpng15.so.15 /usr/local/lib/libpng.so && \
+    ln -s /usr/lib64/libtiff.so.5 /usr/local/lib/libtiff.so
+
+# Update in bashrc
+RUN echo "source /opt/miniconda/etc/profile.d/conda.sh" >> /root/.bashrc && \
+    echo "conda deactivate" >> /root/.bashrc
+
+# Set proxy
+ENV HTTPS_PROXY http://172.16.1.135:3128
+ENV HTTP_PROXY http://172.16.1.135:3128
+
+# Prepare py-3.6 conda env
+RUN . /opt/miniconda/etc/profile.d/conda.sh && \
+    conda create -n py-3.6 python=3.6 -y && \
+    conda activate py-3.6 && \
+    pip install --upgrade pip setuptools wheel && \
+    pip install torch==1.8.0 torchvision==0.9.0  && \
+    pip install mmcv==1.7.0 && \
+    pip cache purge
+
+# Prepare py-3.7 conda env
+RUN . /opt/miniconda/etc/profile.d/conda.sh && \
+    conda create -n py-3.7 python=3.7 -y && \
+    conda activate py-3.7 && \
+    pip install --upgrade pip setuptools wheel && \
+    pip install torch==1.8.0 torchvision==0.9.0  && \
+    pip install mmcv==1.7.0 && \
+    pip cache purge
+
+# Prepare py-3.8 conda env
+RUN . /opt/miniconda/etc/profile.d/conda.sh && \
+    conda create -n py-3.8 python=3.8 -y && \
+    conda activate py-3.8 && \
+    pip install --upgrade pip setuptools wheel && \
+    pip install torch==1.8.0 torchvision==0.9.0  && \
+    pip install mmcv==1.7.0 && \
+    pip cache purge
+
+# Prepare py-3.9 conda env
+RUN . /opt/miniconda/etc/profile.d/conda.sh && \
+    conda create -n py-3.9 python=3.9 -y && \
+    conda activate py-3.9 && \
+    pip install --upgrade pip setuptools wheel && \
+    pip install torch==1.8.0 torchvision==0.9.0  && \
+    pip install mmcv==1.7.0 && \
+    pip cache purge
+
+# Prepare py-3.10 conda env
+RUN . /opt/miniconda/etc/profile.d/conda.sh && \
+    conda create -n py-3.10 python=3.10 -y && \
+    conda activate py-3.10 && \
+    pip install --upgrade pip setuptools wheel && \
+    pip install torch==1.11.0 torchvision==0.12.0  && \
+    pip install mmcv==1.7.0 && \
+    pip cache purge
```

Comment:
What about `RUN` for-loop, and using `ARG` to define mmcv version?

Reply:
Loop added in https://github.com/openxrlab/xrprimer/pull/36/commits/df0a6792f11eb6a4e25a933d859623c882e25b91

Reply:
ARG added in [1a471be](https://github.com/openxrlab/xrprimer/pull/36/commits/1a471be99215559e80ce483ade7d140e15173b16)